recipe-box-frontend/src/main/frontend/src/components/RecipesList/Categories.tsx

```diff
@@ -7,22 +8,109 @@ export default function Categories({
+                    }
+                    onClick={() => handleMealTypeChange("Breakfast")}
+                >
+                    <span className="button__text">Breakfast</span>
+                    <span className="carpator_si_cutit"></span>
+                    <span className="ceasca"></span>
+                    <span className="oala_deschisa"></span>
+                    <span className="sare"></span>
+                </button>
+
+                <button
+                    className={
+                        selectedMealType === "Lunch"
+                            ? "active button"
+                            : "button"
+                    }
+                    onClick={() => handleMealTypeChange("Lunch")}
+                >
+                    <span className="button__text">Lunch</span>
+                    <span className="carpator_si_cutit"></span>
+                    <span className="ceasca"></span>
+                    <span className="oala_deschisa"></span>
+                    <span className="sare"></span>
+                </button>
+
+                <button
+                    className={
+                        selectedMealType === "Dinner"
+                            ? "active button"
+                            : "button"
+                    }
+                    onClick={() => handleMealTypeChange("Dinner")}
+                >
+                    <span className="button__text">Dinner</span>
+                    <span className="carpator_si_cutit"></span>
+                    <span className="ceasca"></span>
+                    <span className="oala_deschisa"></span>
+                    <span className="sare"></span>
+                </button>
+
+                <button
+                    className={
+                        selectedMealType === "Dessert"
+                            ? "active button"
+                            : "button"
+                    }
+                    onClick={() => handleMealTypeChange("Dessert")}
+                >
+                    <span className="button__text">Dessert</span>
+                    <span className="carpator_si_cutit"></span>
+                    <span className="ceasca"></span>
+                    <span className="oala_deschisa"></span>
+                    <span className="sare"></span>
+                </button>
+
+                <button
+                    className={
+                        selectedMealType === "Snack"
+                            ? "active button"
+                            : "button"
+                    }
+                    onClick={() => handleMealTypeChange("Snack")}
+                >
+                    <span className="button__text">Snack</span>
+                    <span className="carpator_si_cutit"></span>
+                    <span className="ceasca"></span>
+                    <span className="oala_deschisa"></span>
+                    <span className="sare"></span>
+                </button>
```

Comment:
Lets try to clean up. You can refactor by creating an array with the categories and map over the array to dynamically create the buttons